core/src/main/kotlin/com/exactpro/th2/read/db/impl/grpc/DataBaseReaderGrpcServer.kt

```diff
@@ -141,5 +147,11 @@ class DataBaseReaderGrpcServer(
 
     companion object {
         private val LOGGER = KotlinLogging.logger { }
+
+        internal fun DbPullResetState.toResetState(): ResetState = ResetState(
+            if(hasAfterDate()) afterDate.toInstant() else null,
+            //TODO: maybe use another protobuf type or check date unit
+            if(hasAfterTime()) LocalTime.ofInstant(afterTime.toInstant(), ZoneOffset.UTC) else null
```

Comment:
Unfortunately, there is no better suitable type in protobuf
https://protobuf.dev/reference/protobuf/google.protobuf/
But I suggest we validate that the date part is 01.01.1970 so the only part that matters is time. We can report an error if the date is also specified

Reply:
What do you mean under `report an error` log error or throw an exception?
I think it shouldn't be critical problem and we should log warn when user pass full date.

Reply:
It is up to you. But I would prefer the exception so user does not expect that the date part is taking into account